packages/admin-manifest/src/index.js

```diff
@@ -0,0 +1,157 @@
+function addManifest( manifest ) {
+	const link = document.createElement( 'link' );
+	link.rel = 'manifest';
+	link.href = 'data:application/manifest+json,' + JSON.stringify( manifest );
```

Comment:
This is clever. The PWA plugin uses the REST API to serve the manifest, but using a `data:` URI simplifies things. Was there a specific reason for doing it inline?

Reply:
Right, this simplifies things and allows us to dynamically generated the icon (for which I'm changing the background color based on the admin color scheme).

Reply:
I guess it's still preferable to use the REST API for the frontend web app manifest, because there's a performance hit for running script with each page load.